### PR TITLE
📖 Improve docs and add license

### PR DESCRIPTION
**Pull request by rubenmoya**
This PR improves the documentation and also adds a license file.

⚠️ Be aware that we're changing the project name to `Acid Swarm`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,63 @@
...
+
+```sh
+DOMAIN_NAME={your-domain} docker stack deploy -c swarm_init/docker-compose.yml infrastructure
 ```
+
+## Contributing
+
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
```
Comment (ghost) [inline README.md:59]: What tests? 😂

File: README.md [hunk 2]
```diff
@@ -1,30 +1,63 @@
...
+
+To monitorize the Docker Swarm cluster we use [Swarmpit](https://swarmpit.io/), it allows us to see in which nodes the applications are deployed, check the logs for each application and the resources used by each node. It also give us some more functionality, like deploy stacks, add an image registry, docker secrets, etc, but we just use it for monitoring right now.
+
+Once the stack is deployed, you can access it in `swarmpit.yourdomain.com`.
+
+### Traefik
+
+To redirect the traffic to the correct container we use [Traefik](https://traefik.io/). With Traefik you can configure a set of rules to determine what petition goes to what container. This is configured directly in the docker-compose files and it continously updates it's configuration so restarts are not needed.
+
+Once the stack is deployed, you can check those rules in `swarmpit.yourdomain.com`.
```
Comment (ghost) [inline README.md:25]:
```suggestion
Once the stack is deployed, you can check those rules in `traefik.yourdomain.com`.
```

File: README.md [hunk 3]
```diff
@@ -1,30 +1,63 @@
...
-# Usage
+You could also use our [Terraform template](https://github.com/acidtango/terracid_tango/tree/with_docker)
+to deploy a Docker Swarm cluster to AWS.
 
-All the following commands must be executed in a Swarm Manager node.
+## What's inside
+
+### Swarmpit
+
+To monitorize the Docker Swarm cluster we use [Swarmpit](https://swarmpit.io/), it allows us to see in which nodes the applications are deployed, check the logs for each application and the resources used by each node. It also give us some more functionality, like deploy stacks, add an image registry, docker secrets, etc, but we just use it for monitoring right now.
```
Comment (ghost) [inline README.md:17]: I think we could remove the last sentence.
```suggestion
To monitorize the Docker Swarm cluster we use [Swarmpit](https://swarmpit.io/), it allows us to see in which nodes the applications are deployed, check the logs for each application and the resources used by each node. It also give us some more functionality, like deploy stacks, add an image registry and manage docker secrets.
```

File: README.md (hunk 1 above)
Comment (rubenmoya) [reply README.md:59]: The ones we have to avoid copy-paste, it looks like they're failing tho

**Pull request closed without merging**